datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -122,7 +134,12 @@ void cleanup() {
         }
 
         if (null != amazonS3) {
-            removeObjectFromS3(amazonS3, amazonBucket, bucketPath);
+            String[] buckets = amazonBucket.split(",");
+            if (buckets.length > 1) {
```

Comment:
`if` is not necessary
Simply 
```
Stream.of(buckets).forEach(it -> removeObjectFromS3(amazonS3, it, bucketPath));
```
will work fine

Reply:
ok